.storybook/config.tsx

```diff
@@ -1,5 +1,22 @@
-import { configure, addDecorator } from "@storybook/react";
+import { configure, addDecorator, addParameters } from "@storybook/react";
 import themeDecorator from "./theme-decorator";
+import { create } from "@storybook/theming";
+
+const theme = create({
+  base: "dark",
+  appBg: "#1f2629",
```

Comment:
Can we bring these hex in from `/lib/theme.js` ?

Reply:
Yeah we could but we don't have a full theme file yet. I could change that in another PR or add the theme to this one

Reply:
I think itd make sense to add those colors to the theme now. In general we can go adding to the theme as we use the values in the components etc. it doesnt always have to be "theme first"